WordPress/Sniffs/NamingConventions/ValidFunctionNameSniff.php

```diff
@@ -133,7 +133,50 @@ protected function processTokenWithinScope(PHP_CodeSniffer_File $phpcsFile, $sta
 
     }//end processTokenWithinScope()
 
+	/**
+	 * Returns the name of the class that the specified class implements.
+	 *
+	 * Returns FALSE on error or if there is no implemented class name.
+	 *
+	 * @param int $stackPtr The stack position of the class.
+	 * @param PHP_CodeSniffer_File $phpcsFile The stack position of the class.
+	 *
+	 * @see PEAR_Sniffs_NamingConventions_ValidFunctionNameSniff::findExtendedClassName()
+	 *
+	 * @todo This needs to be upstreamed and made part of PHP_CodeSniffer_File.
+	 *
+	 * @return string
+	 */
+	public function findImplementedClassName( $stackPtr, $phpcsFile ) {
```

Comment:
I renamed it in 0c1a73e.

> Also, this could be moved to the base `WordPress_Sniff` class so it will be available to our other sniffs in the future.

Actually, the inclusion here is just a stop gap until we can get the method added directly to `PHP_CodeSniffer_File` upstream.